srcalc/src/main/java/gov/va/med/srcalc/service/DefaultAdminService.java

```diff
@@ -31,6 +31,14 @@
     private final RuleDao fRuleDao;
     private final ProcedureDao fProcedureDao;
     
+    /**
+     * Constructs an instance.
```

Comment:
We don't need to document every DAO individually, but we should have at least a summary comment covering all of them.